pkg/kube/inject/template.go

```diff
@@ -27,6 +27,7 @@ import (
 	"google.golang.org/protobuf/types/known/durationpb"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kstrings "k8s.io/utils/strings"
```

Comment:
nit: Do we really need a new import for a 5 line string util func?

https://github.com/kubernetes/utils/blob/24370beab75816ab1d2b33fba9e288ee667530dd/strings/strings.go#L26